dashi/rnaseqc/over_time.py

```diff
@@ -132,44 +315,102 @@ def create_subplot(rna_df, graph_list):
                 dcc.Dropdown(
                     id='graphs_to_plot',
                     multi=True,
-                    options=[{'label': x, 'value': x} for x in graphs_to_plot],
-                    value=graphs_to_plot[:4]
+                    options=[
+                        {'label': x, 'value': x} for x in METRICS_TO_GRAPH
+                    ],
+                    value=METRICS_TO_GRAPH[:4]
                 ),
+                html.Br(),
+                html.Div([
+                    html.Div([
+                        html.Label('Colour by:'),
+                        dcc.Dropdown(
+                            id='colour_by',
+                            options=[
+                                {'label': x['name'], 'value': x['id']} for x
+                                in SHAPE_COLOUR_COLUMN
+                            ],
+                            value='Study Title'
+                        ),
+                    ], style={'width': '45%', 'display': 'inline-block', }),
+                    html.Div([
+                        html.Label('Shape by:'),
+                        dcc.Dropdown(
+                            id='shape_by',
+                            options=[
+                                {'label': x['name'], 'value': x['id']} for x
+                                in SHAPE_COLOUR_COLUMN
+                            ],
+                            value='geo_prep_kit'
+                        ),
+                    ], style={
+                        'width': '45%', 'display': 'inline-block',
+                        'float': 'right'
+                    }),
+                ])
             ], style={'margin': '23px'})]
         ),
-        sd_material_ui.RaisedButton(id='filter_button', label='Filters'),
+        html.Div([
+            html.Div(
+                sd_material_ui.RaisedButton(
+                    id='filter_button', label='Options'
+                ), style={'display': 'inline-block', }
+            ),
+            html.Div(
+                sd_material_ui.RaisedButton(
+                    id='update_from_table_button', label='Update Order',
+                ), style={'display': 'inline-block', 'margin-left': '15px'}
+            ),
+        ]),
     ]),
-    dcc.Loading(id="graph_loader", children=[
+    dcc.Loading(id="graph_loader_plot", children=[
         sd_material_ui.Paper(
             [dcc.Graph(
                 id='graph_subplot',
             )]
         ),
+    ]),
+    dcc.Loading(id="graph_loader_data_table", children=[
+        sd_material_ui.Paper(
+            [dash_table.DataTable(
+                id='data_table',
+                columns=DEFAULT_TABLE_COLUMN + [
+                    {'name': i, 'id': i} for i in METRICS_TO_GRAPH[:4]
+                ],
+                data=RNA_DF.to_dict('records'),
+                page_size=50,
+                sort_action='native',
+                sort_mode='multi',
+                export_format='csv'
```

Comment:
Is it possible to name the downloaded sheet? `Data` isn't very informative.

Reply:
Not at this point. I have made a ticket https://github.com/plotly/dash-table/issues/575